packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/mgt-file-upload.scss

```diff
@@ -234,9 +236,58 @@ $file-upload-border-drag: var(--file-upload-border-drag, 1px dashed #0078d4);
   }
 }
 
-[dir="rtl"] {
-  :host .file-upload-status{
+[dir='rtl'] {
+  :host .file-upload-status {
     left: 0;
     right: 28px;
   }
 }
+
+@media (forced-colors: active) {
```

Comment:
Blocking comment to prevent merging until a11y sign off is received

Reply:
removing blocking comment in light of consistent behavior with fluent web components